### DISCUSSION TITLE
Device Test Tool Media1 streaming "The stream Uri has incorrect type. Expected type is IPv4"

### DISCUSSION THREAD

**Discussion by pawelbialek** [onvif/testspecs#86] [Q&A] [answered]
I am receiving the error in the title when we are attempting to test our ipv6 support implementation.
The implementation detects if requests were made using ipv4 or ipv6 and makes replies according to this.
Any other tests in the `Diagnostics` tab, such as ones that test `GetSnapshotUri` work well with this and react to this correctly.
Only tests from RTSS-1-1 (video streaming), RTSS-2-1 (audio streaming) & RTSS-6-1 (audio backchannel) show this discrepancy.

In ONVIF caps, used for instance in device service `GetServices` API, we point out that our implementation is ipv6-capable by setting the `IPVersion6` flag to true.
But this seems to now do the job.

Is there some additional configuration to be set within ONVIF or ODDT that may change the behavior ?
I doubt the standard imposes ipv4 on all stream uris, it is just we are doing something wrong.

Thank you in advance!

**Comment by pawelbialek** [comment]
Almost two months without an answer ...

**Comment by madhurao68** [comment, marked as answer]
Hi,
By default, DTT starts in IPv4 though some tests support IPv6. Please note that DTT does not support IPv6 conformance. Apologies for the delayed response.
